webpack-stats.json

```diff
@@ -1 +1 @@
-{"status":"done","chunks":{"main":[{"name":"main.js","path":"/home/liubomyr/newproject/events/event_manager/events/static/bundles/main.js"}]}}
\ No newline at end of file
+{"status":"done","chunks":{"main":[{"name":"main.js","path":"/home/denis/Pythons/event_manager/events/static/bundles/main.js"}]}}
```

Comment:
this file as also 'main.js' should not be added to commit. Please delete them and add to .gitignore